packages/net-misc/remmina/remmina.exlib

```diff
@@ -36,18 +41,19 @@ CMAKE_SRC_CONFIGURE_PARAMS=(
     -DWITH_GCRYPT=ON
     -DWITH_GETTEXT=ON
     -DWITH_LIBSECRET=ON
-    -DWITH_LIBSSH=ON
     -DWITH_TRANSLATIONS=ON
-    -DWITH_VTE=ON
-    -DWITH_ZLIB=ON
+    -DWITH_FREERDP=ON
+    -DWITH_LIBVNCSERVER=ON
 
     # off
     -DWITH_APPINDICATOR=OFF
+    -DWITH_TELEPATHY=OFF
```

Comment:
again, you did something to the alphabetical ordering

Reply:
You got me here, I left some mess after editing. It should be fixed now.